### PR TITLE
Adding Rubocop.yml

### PR DESCRIPTION
**Pull request by eddierubeiz**
It's a pretty defanged version right now - it only looks at app, and a bunch of the more annoying cops are turned off. Feel free to adjust the settings; this is just meant as a starting point.

### CODE REVIEW

**Comment by jrochkind** [comment]
What's the thinking behind restricting to app/?

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Layout:
+  EnforcedStyle: rails
+
+Style/CollectionMethods:
+  PreferredMethods:
+    collect: 'map'
+    collect!: 'map!'
+    inject: 'reduce'
+    detect: 'find'
+    find_all: 'select'
```
Comment (jrochkind) [inline .rubocop.yml]: i always use `collect` and `find_all`. Maybe I should finally join everyone else?

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    - db/migrate/**/*
+    - kithe.gemspec
+    - Rakefile
+    - Gemfile
+    - bin/rails
+    - spec/**/*
+    - test/**/*
+    - config/**/*
+    - lib/**/*
+    # - app/**/*
```
Comment (jrochkind) [inline .rubocop.yml]: let's include lib/ too

Comment (eddierubeiz) [reply .rubocop.yml]: Cool, added lib. Not sure what should and shouldn't be checked at this early stage and didn't want to flood the codebase with errors in generated code.

**Comment by jrochkind** [comment]
What do we have to do so rubocop actually runs? On travis? When we run tests manually?

**Comment by eddierubeiz** [comment]
I was using it as a git commit hook using a slightly adapted version of https://gist.github.com/stevenharman/f5fece7fd39356e57ce3c710911f714f .

**Comment by jrochkind** [comment]
Hmm. I'm not sure if I want to use a git commit hook, but I can try!

To merge, I think we need at minimum: Instructions added to README on how to add as git commit hook, or how to run manually. 

We might want to get it to run automatically by travis, so PR's can't be merged unless they pass? What do you think? I kind of think this thing is unlikely to be complied with unless it's enforced somehow (and there's no way to enforce individual git commit hooks). Making sure travis runs it will enforce it on PR's, although not direct commits to master, which I'm still doing sometimes. 

Or maybe it's okay to just have it there, without enforcement? I dunno. What are your thoughts? Even for that, we need instructions in the README though.

**Comment by jrochkind** [comment]
I'm going to close this for now, because apparently we're not going to do it for now!

**Comment by eddierubeiz** [comment]
Oh yeah. This has been stale for a while.

**Pull request closed without merging**